### ISSUE TITLE
Unable to use BitmapIcon for NavigationViewItem in Mac and Linux

### ISSUE THREAD

**Issue by LeftTwixWand** [unoplatform/uno#4708]
## Current behavior
I'm trying to use custom .png as icon for `NavigationViewItem`:

  ```XAML
<NavigationView>
        <NavigationView.MenuItems>
             <NavigationViewItem >
                <NavigationViewItem.Icon>
                    <BitmapIcon UriSource="ms-appx:///Assets/Icons/myIcon.png"/>
                    <!-- Or this <BitmapIcon UriSource="Assets/Icons/myIcon.png"/> -->
                </NavigationViewItem.Icon>
            </NavigationViewItem>
        <NavigationView.MenuItems>
 <NavigationView>
```

For UWP it works, but at Linux and Mac I've got this error:  `BitmapIcon doesn't contains definition for 'UriSource' (CS0117)`

## Environment
Nuget Package:
- [x] Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia
- [ ] Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia
- [ ] Uno.SourceGenerationTasks
- [ ] Uno.UI.RemoteControl / Uno.WinUI.RemoteControl
- [ ] Other: 

Nuget Package Version(s):
3.3.0

Affected platform(s):

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [x] macOS
- [x] Skia
  - [ ] WPF
  - [x] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

IDE:

- [ ] Visual Studio 2017 (version: )
- [ ] Visual Studio 2019 (version: )
- [x] Visual Studio for Mac (version: )
- [ ] Rider Windows (version: )
- [ ] Rider macOS (version: )
- [x] Visual Studio Code for Linux (version: latest)

**Comment by LeftTwixWand** [comment]
So, it's not supported now?

**Comment by jeromelaban** [comment]
Thanks for the report.

Could you provide a sample solution which demonstrates the issue ?

Here's the BitmapIcon source file, which contains UriSource:
https://github.com/unoplatform/uno/blob/e53bfaa4675447e6c470faeb6c0d90f67b616521/src/Uno.UI/UI/Xaml/Controls/BitmapIcon.cs#L92

**Comment by LeftTwixWand** [comment]
It's super strange, but in empty project it works. 
All the nuget packages and dependencies are similar...

**Comment by LeftTwixWand** [comment]
Thanks, I'll try to find a problem and reopen this issue later, to provide more information.

**Comment by LeftTwixWand** [comment]
Hi. After a few days of searching I finally understand what's wrong.
I've rolled back all files, but forgot about references in Mac and Linux projects.

I'm also using this libraries:
`Microsoft.Toolkit.Mvvm` version 7.0.0-preview4
`Uno.Microsoft.Toolkit.Uwp.UI.Controls` version 6.1.0-build.201.gdd585882df

And as I anderstand: `Uno.Microsoft.Toolkit.Uwp.UI.Controls` at the build time replaced default `Uno.UI` and `Uno.UI.RemoteControl` libraryies. As result - error, because Toolkit library has no definition of `UriSource` property.

**Comment by jeromelaban** [comment]
Thanks for the update. What's still missing in this issue ? Can you still provide a repro project ?

**Comment by LeftTwixWand** [comment]
You can find a sample repository [here](https://github.com/LeftTwixWand/UnoApp).

**Comment by jeromelaban** [comment]
Thank you for the repro.

This was fixed by https://github.com/unoplatform/uno/pull/4561 and will be avalable in the next Uno release.

**Comment by LeftTwixWand** [comment]
> Thank you for the repro.
> 
> This was fixed by #4561 and will be avalable in the next Uno release. You can update to the latest dev packages to try it out.

Thanks!
